uber/models/attendee.py

```diff
@@ -758,40 +759,90 @@ def donation_swag(self):
         return donation_items + extra_donations
 
     @property
-    def merch(self):
+    def merch_items(self):
```

Comment:
We now have both the existing ``Attendee.merch`` property which is the same as before and a new ``Attendee.merch_items`` property which returns a list of those items.  This makes it easier for the backend to pass something to the frontend for nice formatting.